src/main/java/net/authorize/sample/TransactionReporting/GetAccountUpdaterJobDetails.java

```diff
@@ -0,0 +1,177 @@
+		String reFid = "123456";
+		Paging paging = new Paging();
+		paging.setLimit(100);
+		paging.setOffset(2);
+
+		// Create the API request and set the parameters for this specific request
+		GetAUJobDetailsRequest apiRequest = new GetAUJobDetailsRequest();
+		apiRequest.setMerchantAuthentication(merchantAuthenticationType);
+		apiRequest.setPaging(paging);
+		apiRequest.setRefId(reFid);
+		apiRequest.setMonth(month);
+		apiRequest.setModifiedTypeFilter(AUJobTypeEnum.ALL);
+
+		// Call the controller
+		GetAUJobDetailsController controller = new GetAUJobDetailsController(apiRequest);
+		controller.execute();
+
+		GetAUJobDetailsResponse response = new GetAUJobDetailsResponse();
+		response = controller.getApiResponse();
+
+		// If API Response is OK, go ahead and check the transaction response
+		if (response != null && response.getMessages().getResultCode() == MessageTypeEnum.OK) {
+
+			System.out.println("SUCCESS: Get Account Updater job details for Month and year : " + month);
+
+			if (response.getAuDetails() == null) {
+				System.out.println("No GetAccountUpdaterjobdetails for this month and year.");
+				return response;
+			}
+
+			ArrayList<AuUpdateType> updateTypeList = new ArrayList<AuUpdateType>();
+			ArrayList<AuDeleteType> deleteTypeList = new ArrayList<AuDeleteType>();
+
+			for (AuDetailsType details : response.getAuDetails().getAuUpdateOrAuDelete()) {
+
+				System.out.println("---Customer profile details Start---");
+				System.out.println("customerProfileID:" + details.getCustomerProfileID());
+				System.out.println("customerPaymentProfileID:" + details.getCustomerPaymentProfileID());
+				System.out.println("customerFirstname:" + details.getFirstName());
+				System.out.println("customerLastname:" + details.getLastName());
+				System.out.println("updateTimeUTC:" + details.getUpdateTimeUTC());
+				System.out.println("reasonCode:" + details.getAuReasonCode());
+				System.out.println("reasonDescription:" + details.getReasonDescription());
+
+				if (details.getClass().getTypeName().toString().contains("AuUpdateType")) {
+
+					updateTypeList.add((AuUpdateType) details);
+
+				} else if (details.getClass().getTypeName().toString().contains("AuDeleteType")) {
+
+					deleteTypeList.add((AuDeleteType) details);
+
+				}
+
+				if (!(updateTypeList.isEmpty())) {
+					for (int i = 0; i < updateTypeList.size(); i++) {
+
+						System.out.println("---AU Update Start---");
+						System.out.println("customerProfileID:" + details.getCustomerProfileID());
+						System.out.println("customerPaymentProfileID:" + details.getCustomerPaymentProfileID());
+						System.out.println("customerFirstname:" + details.getFirstName());
+						System.out.println("customerLastname:" + details.getLastName());
+						System.out.println("updateTimeUTC:" + details.getUpdateTimeUTC());
+						System.out.println("reasonCode:" + details.getAuReasonCode());
+						System.out.println("reasonDescription:" + details.getReasonDescription());
```

Comment:
Please check the Java sample codes for the strings to print here. For example, it should print "Profile ID", and NOT "customerProfileID".

Reply:
Changes done as per suggestion.